activestorage/app/models/active_storage/variant.rb

```diff
@@ -108,14 +108,13 @@ def process
       end
     end
 
-
     def specification
       @specification ||=
         if ActiveStorage.web_image_content_types.include?(blob.content_type)
           Specification.new \
             filename: blob.filename,
             content_type: blob.content_type,
-            format: nil
+            format: Rack::Mime::MIME_TYPES.invert[blob.content_type]
```

Comment:
Since Active Storage uses [MimeMagic](https://github.com/minad/mimemagic) (by way of [Marcel](https://github.com/basecamp/marcel)), you might consider using it here:

```suggestion
            format: MimeMagic.new(blob.content_type).extensions.first
```

Reply:
@jonathanhefner Changed - still digging into the rails codebase so thanks for the suggestion, make sense to use `MimeMagic` since it is already used by Active Storage 👍